reviews/methods1.Rmd

```diff
@@ -0,0 +1,450 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+
+All changes of the manuscript since this review can be viewed on GitHub:
+
+<https://github.com/aaronpeikert/bayes-prereg/compare/v0.2.0...main#diff-a9a4aad3fa8c9c10c5404b632bc3a01a25d2d8430eb932bc35c76769963e4b70>
+
+Our responses to the review are formatted as follows.
+
+Normal text = the reviewer's comment
+
+> Indented text = our response
+
+> *Indented italicized = quote from the revised paper*
+
+
+Reviewer #1: The authors set out to explain what the primary aim of preregistration is.
+I think they are correct that many scientists have hardly thought about this question.
+This is not in itself bad - preregistration intuitively makes sense - but it is important to clearly articulate what the aim of preregistration is.
+
+> We agree that any tool applied thoughtlessly will seldom be effective.
+> A clear aim of preregistration could be helpfull to shape its application, especially given that the scientific comunity is still in an experimenting phase around central issues of preregistration, e.g., what level of detail a preregistration should have, under what circumstances to deviate from it, how to review it, or how to deal with exploratory research questions.
```

Comment:
I simply would put "we agree". 
There is no reason to start new discussions on the few points we agree upon.

Reply:
Ok